Source/Core/CesiumTerrainProvider.js

```diff
@@ -489,17 +493,16 @@ define([
      * @param {Number} x The X coordinate of the tile for which to request geometry.
      * @param {Number} y The Y coordinate of the tile for which to request geometry.
      * @param {Number} level The level of the tile for which to request geometry.
-     * @param {Boolean} [throttleRequests=true] True if the number of simultaneous requests should be limited,
-     *                  or false if the request should be initiated regardless of the number of requests
-     *                  already in progress.
+     * @param {Request} [request] The request object.
+     *
      * @returns {Promise.<TerrainData>|undefined} A promise for the requested geometry.  If this method
      *          returns undefined instead of a promise, it is an indication that too many requests are already
      *          pending and the request will be retried later.
      *
      * @exception {DeveloperError} This function must not be called before {@link CesiumTerrainProvider#ready}
      *            returns true.
      */
-    CesiumTerrainProvider.prototype.requestTileGeometry = function(x, y, level, throttleRequests) {
+    CesiumTerrainProvider.prototype.requestTileGeometry = function(x, y, level, request) {
```

Comment:
I think I've been pretty clear that I'm squarely in the "don't throttle" crowd (I'm all for prioritization and cancellation, just not throttling) so take this with that bias up front.  

Just because a server supports HTTP/2, doesn't mean the client does.  So it's impossible for the developer to make this decision during development.  As far as I know, it's also impossible to detect at runtime whether you're using HTTP/2, so we can't do some magic on our end either.  That means the best option is to not throttle at all and let the browser handle it.

I tried to hunt around to see what other webmaps are doing, and as far as I can tell none of them throttle (I checked OL/Leaflet/Mapbox).  OpenLayers has an open issue to cancel unneeded requests, but that's it.

But since I'm not going to be able to convince you guys to not do it, I think the second best option is having a single top-level setting on `RequestScheduler` to disable it.  I know the initial implementation had this, I'm not sure about the one in this branch (haven't gotten to that part of the code yet).